### ISSUE TITLE
using sum of marginal likelihoods of different datasets as loss function

### ISSUE THREAD

**Issue by tiger314** [cornellius-gp/gpytorch#1718]
I have multiple datasets $(X_i, y_i)_{i=1}^{n}$ and I would like to use the sum of the marginal likelihoods of each dataset as my loss function to learn a shared set of GP hyperparameters (so not one set of hyperparameters per dataset but a single set of hyperparameters across all datasets). Is there a way I can do this? Thanks!

**Comment by jacobrgardner** [comment]
Yep, this is pretty simple using something like https://github.com/cornellius-gp/gpytorch/blob/master/examples/08_Advanced_Usage/Simple_Batch_Mode_GP_Regression.ipynb

Any parameters you want to share just don't specify batch shapes for and it'll just learn one of them. You can probably also do the same thing via our batch independent multi output GP example notebook.
